sdks/wasm/framework/src/System.Net.Http.WebAssemblyMessageHandler/WebAssemblyMessageHandler.cs

```diff
@@ -1,48 +1,69 @@
 using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Net;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
+using WebAssembly;
 using WebAssembly.Core;
 using WebAssembly.Host;
 
-namespace WebAssembly.Net.Http.HttpClient {
-	public class WasmHttpMessageHandler : HttpMessageHandler {
+namespace System.Net.Http {
+	public class WebAssemblyMessageHandler : HttpMessageHandler {
```

Comment:
Please change the name to `WebAssemblyHttpHandler` to align with existing convention (https://docs.microsoft.com/en-us/dotnet/api/system.net.http.winhttphandler?view=dotnet-plat-ext-3.1)

Also change the package name to reflect that